src/explore-education-statistics-admin/src/pages/admin-dashboard/components/PublicationsTab.tsx

```diff
@@ -121,22 +99,20 @@ const PublicationsTab = ({ isBauUser }: Props) => {
       {isBauUser && (
         <>
           {themes && themes.length > 0 && (
-            <FormThemeTopicSelect
-              id="publicationsReleases-themeTopic"
-              legend="Choose a theme and topic to view publications for"
+            <FormThemeSelect
```

Comment:
I've commented on this component file itself about maybe either keeping this bespoke select component or whether to just use FormSelect instead - up to you :)